crates/cast/src/starknet_commands/account/delete.rs

```diff
@@ -0,0 +1,114 @@
+use anyhow::{anyhow, bail, Context, Result};
+use camino::Utf8PathBuf;
+use cast::helpers::response_structs::AccountDeleteResponse;
+use cast::helpers::scarb_utils::get_scarb_manifest;
+use clap::Args;
+use promptly::prompt;
+use serde_json::Map;
+use std::fs::File;
+use std::io::Read;
+use std::io::Write;
+
+#[derive(Args, Debug)]
+#[command(about = "Delete account information from the accounts file")]
+pub struct Delete {
+    /// Name of the account to be deleted
+    #[clap(short, long)]
+    pub name: Option<String>,
+
+    /// If passed with false, Scarb profile won't be removed
+    #[clap(long, num_args = 1, default_value = "true")]
+    pub delete_profile: Option<bool>,
+
+    /// Network where the account exists; defaults to network of rpc node
+    #[clap(long)]
+    pub network: Option<String>,
+}
+
+#[allow(clippy::too_many_arguments)]
+pub fn delete(
+    name: &str,
+    path: &Utf8PathBuf,
+    path_to_scarb_toml: &Option<Utf8PathBuf>,
+    delete_profile: Option<bool>,
+    network_name: &str,
+) -> Result<AccountDeleteResponse> {
+    let contents = std::fs::read_to_string(path.clone()).context("Couldn't read accounts file")?;
+    let items: serde_json::Value = serde_json::from_str(&contents)
+        .map_err(|_| anyhow!("Failed to parse accounts file at {path}"))?;
+
+    if items[&network_name].is_null() {
+        bail!("No accounts defined for network {}", network_name);
+    }
+    if items[&network_name][&name].is_null() {
+        bail!("Account with name {name} does not exist")
+    }
+
+    let mut items: Map<String, serde_json::Value> =
+        serde_json::from_str(&contents).expect("failed to read file { path }");
+
+    // Let's ask confirmation
+    let prompt_text =
+        format!("Do you want to remove account {name} from network {network_name}? (Y/n)");
```

Comment:
This implies the account will be removed from network, and we only remove it from local file. Let's change to something like:
```suggestion
        format!("Do you want to remove the account {name} deployed to network {network_name} from local file {path}? (Y/n)");
```

wdyt?

Reply:
👍